.asf.yaml

```diff
@@ -1,3 +1,17 @@
+github:
+  enabled_merge_buttons:
+    squash: true
+    merge: false
+    rebase: true
+  labels:
+    - flink
+    - big-data
+    - java
+    - scala
+    - python
+    - sql
```

Comment:
Shall we update the labels here? For example, add `machine learning`, remove `scala` and `sql`?

Reply:
Thanks for the suggestion. Good catch.

I have updated the PR to remove the label additions. The reason is that apache/kafka and apache/flink-statefun do not have `labels` in their `.asf.yaml`. And I also could not find any concrete usage of these labels in apache/flink's pull requests.

The commit message has been updated as appropriate.